ArcDPS Boon Table/main.cpp

```diff
@@ -34,6 +36,26 @@ void writeIni();
 bool modsPressed();
 bool canMoveWindows();
 
+void log_file(char* str);
+void log_arc(char* str);
+
+/* arcdps exports */
+void* filelog;
+void* arclog;
+
+/* log to arcdps.log, thread/async safe */
```

Comment:
As far as i know, the arcdps implementation of it, is kind of thread safe.
Also, why casting it every time?
And we can use `const char*`, cause arcdps is doing nothing with the `char*`
See the killproof.me plugin, how i did it there: https://github.com/knoxfighter/arcdps-killproof.me-plugin/blob/ec28f79bf521148f13986559ff3accdf0bc7a1cb/dllmain.cpp#L424

Reply:
I removed all the logs, as per the other comment. This was part of it. I only used it to log the events.